src/odemis/gui/cont/correlation.py

```diff
@@ -51,6 +52,34 @@ def update_image_in_views(s: StaticStream, views: list) -> None:
             if st is s:
                 sp._shouldUpdateImage()
 
+def convert_rgb_to_sem(rgb_stream: RGBStream) -> StaticSEMStream:
```

Comment:
Hopefully, if data_to_static_streams() is adjusted, this whole part can be dropped.

Reply:
See comment in doc about stitched tilesets